docs/sql-ref-syntax-qry-select-tvf.md

```diff
@@ -21,25 +21,7 @@ license: |
 
 ### Description
 
-A table-valued function (TVF) is a function that returns a relation or a set of rows.
-
-### Syntax
-
-```sql
-function_name ( expression [ , ... ] ) [ table_alias ]
-```
```

Comment:
```[ table_alias ]``` doesn't work with the 2nd type of TVF. Seems it doesn't have much value to have a syntax that only has ```function_name ( expression [ , ... ] )```, so I deleted this section.